Sources/Scotty/Route.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
 //
 
-import Foundation
+import UIKit
```

Comment:
Is this necessary? I don't think the current version has any trouble compiling with the Foundation import.